### PR TITLE
Feature: Notification email

### LINKED ISSUE #5: Notification Email
We have to be able to inform users about **important** notifications by email.

### PR DESCRIPTION
**Pull request by AlirezaT99**
Closes #5 

- TODO:
  - [ ] Email content must look better
  - [ ] Email host user & password must be added to env & secrets @emranbm

### CODE REVIEW

**Comment by emranbm** [comment]
@AlirezaT99 Thanks for the PR. I recommend making it *Ready for review` and creating 2 issues for the mentioned TODOs; as this PR can be reviewed/merged independently.

File: backend/WSS_Site/settings.py [hunk 1]
```diff
@@ -244,3 +244,10 @@
 local_settings_path = os.path.join(os.path.dirname(__file__), 'local_settings.py')
 if os.path.exists(local_settings_path):
     exec(open(local_settings_path, 'rb').read())
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'TODO@EMRAN.MERCI'
```
Comment (emranbm) [inline backend/WSS_Site/settings.py:253]: This is not a secret and can be hardcoded.

File: backend/templates/consts.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+PAYMENT_HEADER='تاییدیه پرداخت رویداد WSS'
+PAYMENT_HTML_CONTENT=(
+    "<body dir=\"rtl\">"
+    "شرکت کننده‌ی محترم، {}\n"
```
Comment (emranbm) [inline backend/templates/consts.py:4]: Does `\n` work in HTML?

**Review by emranbm — CHANGES_REQUESTED** [review]

**Comment by emranbm** [comment]
Have you tested it?

File: backend/WSS_Site/settings.py (hunk 1 above)
Comment (AlirezaT99) [reply backend/WSS_Site/settings.py:253]: Indeed

File: backend/templates/consts.py (hunk 2 above)
Comment (AlirezaT99) [reply backend/templates/consts.py:4]: Not at all. Must be replaced with <br>

**Review by emranbm — APPROVED** [review]

**Pull request merged**